instrumentation/all/Gemfile

```diff
@@ -7,3 +7,5 @@
 source 'https://rubygems.org'
 
 gemspec
+
+gem 'opentelemetry-instrumentation-racecar', path: '../racecar'
```

Comment:
Why is this added here? It isn't necessary for the other instrumentation gems.

Reply:
actually, I'm wrong. This line is required so that the we can add the new instrumentation to all before it has been released. If we remove that line there are build errors as the all gem doesn't have a version of this gem to reference.